docs/s1-uc1-polarization.json

```diff
@@ -81,7 +91,8 @@
           "from_node": "cr"
         },
         "name": "bands",
-        "type": "bands"
+        "type": "bands",
+        "label": 0
```

Comment:
Can't you directly use the label name "B" and then remove the rename_label_cr node?

Reply:
@m-mohr : Yes, good idea, with the new `label` parameter this would be the best solution!

Reply:
Yes agree, I tried that and at the moment this fails at the GEE backend, I think because "add_dimension" does not set the label to the GEE object yet, but just in the JS metadata.

Reply:
Could you make a fix?

Reply:
Yes I am on it right now. Should I make a new pull request, or can I just add it here ?

Reply:
Can be part of this PR, but whatever you prefer.